src/main/java/org/opensearch/performanceanalyzer/reader/SearchBackPressureMetricsSnapShot.java

```diff
@@ -63,43 +70,43 @@ public SearchBackPressureMetricsSnapShot(Connection conn, Long windowStartTime)
                                                 AllMetrics.SearchBackPressureStatsValue
                                                         .SEARCHBP_SHARD_STATS_RESOURCE_HEAP_USAGE_CANCELLATIONCOUNT
                                                         .toString()),
-                                        Long.class));
+                                        Integer.class));
                         this.add(
                                 DSL.field(
                                         DSL.name(
                                                 AllMetrics.SearchBackPressureStatsValue
                                                         .SEARCHBP_SHARD_STATS_RESOURCE_HEAP_USAGE_CURRENTMAX
                                                         .toString()),
-                                        Long.class));
+                                        Integer.class));
                         this.add(
                                 DSL.field(
                                         DSL.name(
                                                 AllMetrics.SearchBackPressureStatsValue
                                                         .SEARCHBP_SHARD_STATS_RESOURCE_HEAP_USAGE_ROLLINGAVG
                                                         .toString()),
-                                        Long.class));
+                                        Integer.class));
                         this.add(
                                 DSL.field(
                                         DSL.name(
                                                 AllMetrics.SearchBackPressureStatsValue
                                                         .SEARCHBP_SHARD_STATS_RESOURCE_CPU_USAGE_CANCELLATIONCOUNT
                                                         .toString()),
-                                        Long.class));
+                                        Integer.class));
 
                         this.add(
                                 DSL.field(
                                         DSL.name(
                                                 AllMetrics.SearchBackPressureStatsValue
                                                         .SEARCHBP_SHARD_STATS_RESOURCE_CPU_USAGE_CURRENTMAX
                                                         .toString()),
-                                        Long.class));
+                                        Integer.class));
                         this.add(
                                 DSL.field(
                                         DSL.name(
                                                 AllMetrics.SearchBackPressureStatsValue
                                                         .SEARCHBP_SHARD_STATS_RESOURCE_CPU_USAGE_CURRENTAVG
                                                         .toString()),
-                                        Long.class));
+                                        Integer.class));
```

Comment:
The cpu usage will be a floating type? Better to represent it with Long then.

Reply:
Will change all fields with "Average" to long since they would all be floating type